packages/components/src/Combobox/components/ComboboxContent/ComboboxHeader.css

```diff
@@ -0,0 +1,5 @@
+.header {
+  display: flex;
+  padding: 16px 16px 0 16px;
```

Comment:
💅 tokens pls and thank u

Reply:
lol let it be known that I know this - but was quickly putting px's instead of tokens. Still......shame!